src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/form/common/attributes/create-button.js

```diff
@@ -113,9 +134,16 @@ define([
         }
 
         modal.$el.on('click', '.attribute-choice', function () {
+          const dataRoute = $(this).attr('data-route');
+
           modal.close();
           modal.$el.remove();
-          router.redirect($(this).attr('data-route'), {trigger: true});
+
+          analytics.track('attribute:create:type-selected', {
+            name: self.getAttributeType(dataRoute),
```

Comment:
The `name` property is not appropriate.
 ```suggestion
            type: self.getAttributeType(dataRoute),
```